rbx_binary/src/chunk.rs

```diff
@@ -41,7 +41,7 @@ impl Chunk {
                 zstd::bulk::decompress(&compressed_data, header.len as usize)?
             } else {
                 log::trace!("LZ4 compression");
-                lz4::block::decompress(&compressed_data, Some(header.len as i32))?
+                lz4_flex::block::decompress(&compressed_data, header.len as usize).unwrap()
```

Comment:
```suggestion
                lz4_flex::block::decompress(&compressed_data, header.len as usize).map(io::Error::other)
```

We probably don't want to unwrap in the off chance the decompression doesn't succeed. If nothing else, this should be `expect` instead of `unwrap`.